nntrainer/src/flatten_layer.cpp

```diff
@@ -37,24 +37,21 @@ int FlattenLayer::initialize(bool last) {
   return status;
 }
 
-Tensor FlattenLayer::forwarding(Tensor in, int &status) {
-  hidden = Tensor(in.batch(), output_dim.channel(), output_dim.height(),
-                  output_dim.width());
+sharedTensor FlattenLayer::forwarding(sharedTensor in) {
+  input = in.get()[0];
+
+  hidden = Tensor(output_dim);
```

Comment:
I think this should be as it is. This can be differ according to train and inference. Cause currently we are config base data with training. ( Of course this is one of the issues we have to resolve)

Reply:
Fixed for both flatten and pooling layers.